src/components/main/CopyButton.tsx

```diff
@@ -37,7 +37,7 @@ const CopyButton: FC<Props> = ({ itemIds, color, id, type, onClick }) => {
 
   return (
     <GraaspCopyButton
-      type={type}
+      type={type as ActionButtonVariant}
```

Comment:
Is this caused by the nullable? If yes, consider using a non-nullable prop type for type, or returning null if not set (but I think removing the nullable makes sense).